addons/website_forum/models/forum.py

```diff
@@ -197,7 +197,7 @@ class Post(models.Model):
 
     name = fields.Char('Title')
     forum_id = fields.Many2one('forum.forum', string='Forum', required=True)
-    content = fields.Html('Content', strip_style=True)
+    content = fields.Html('Content', sanitize=False)
```

Comment:
Why would you make that change ? I think post contents should be sanitized ? :smile:

Reply:
Yes @qsm-odoo  indeed content should be sanitized, but that time it was not allowing to insert iframe tag for posting videos on forum, but now it is fixed in master, so now can remove this